docker-compose.yaml

```diff
@@ -0,0 +1,37 @@
+version: '3.7'
```

Comment:
3.8 exists I think. But doesn't matter really.

Reply:
Will update it the next time I work on the file. Cba testing it again just for this bump :p